### PR TITLE
test(portrait, preview, profile): add missing tests and refactor FE-2355

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

add missing tests for default portrait, refactor for preview/profile components, remove tests for
portrait classic

<del> - [ ] Release notes (Conventional Commits)  <del>
<del>  - [ ] Unit tests <del>  
- [x] Cypress automation tests

<del>  - [ ] Storybook added or updated <del>  
<del>  - [ ] Theme support <del>  
<del>  - [ ] Typescript `d.ts` file added or updated <del>  

### Testing instructions

run: npm run test-cypress and choose proper feature file

### CODE REVIEW

File: cypress/features/regression/portrait.feature [hunk 1]
```diff
@@ -0,0 +1,97 @@
+Feature: Portrait default component
+	I want to test Portrait default component
```
Comment (DlgSHi) [inline cypress/features/regression/portrait.feature]: to large indents

File: cypress/features/regression/profile.feature [hunk 2]
```diff
@@ -1,9 +1,8 @@
-# This feature require separate task for update
-Feature: Profile component
-  I want to change Profile component properties
+Feature: Profile default component
+  I want to change Profile component default properties
```
Comment (DlgSHi) [inline cypress/features/regression/profile.feature]:
```suggestion
  I want to change default Profile component properties
```

File: cypress/features/regression/portrait.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/portrait.feature]: on my local it's good. I think that is some git issue

**Pull request merged**